setuptools/command/bdist_egg.py

```diff
@@ -262,6 +266,8 @@ def zap_pyfiles(self):
 
                     pattern = r'(?P<name>.+)\.(?P<magic>[^.]+)\.pyc'
                     m = re.match(pattern, name)
+                    # We shouldn't ind any non-pyc files in __pycache__
+                    assert m is not None
```

Comment:
Not strictly an annotation change, but I included it here because it didn't feel worth doing a PR just for this